actors/evm/src/interpreter/instructions/storage.rs

```diff
@@ -0,0 +1,83 @@
+use {
+    crate::interpreter::{ExecutionState, StatusCode, System, U256},
+    fil_actors_runtime::runtime::Runtime,
+    fvm_ipld_blockstore::Blockstore,
+};
+
+#[inline]
+pub fn sload<'r, BS: Blockstore, RT: Runtime<BS>>(
+    state: &mut ExecutionState,
+    platform: &'r System<'r, BS, RT>,
```

Comment:
There's some naming incongruence left over from past refactors. Platform/System/Host... Need to fix this eventually.